faf-commons-data/src/main/java/com/faforever/commons/replay/ReplayDataParser.java

```diff
@@ -375,6 +377,18 @@ private void parseGiveResourcesToPlayer(Map<String, Object> lua) {
     }
   }
 
+
+  void parseModeratorEvent(Map<String, Object> lua) {
+    String messageContent = (String) lua.get("Message");
+    Float fromFloat = (Float) lua.get("From");
+    int activeCommandSource = 0; // activeCommandSource is not available in TestModeratorEvents.fafreplay, yet
+    if (lua.containsKey("activeCommandSource")) {
+      activeCommandSource = ((Number) lua.get("activeCommandSource")).intValue();
+    }
```

Comment:
This is not how it works, but it's also not blocking the pull request. We can look into this the next iteration.